extension/js/common/helpers.ts

```diff
@@ -11,6 +11,10 @@ import { ContactStore } from './platform/store/contact-store.js';
 import { KeyStore } from './platform/store/key-store.js';
 import { PassphraseStore } from './platform/store/passphrase-store.js';
 import { Bm } from './browser/browser-msg.js';
+import { FLAVOR } from './core/const.js';
+export const isEnterpriseBuildUsed = () => {
+  return FLAVOR === 'enterprise';
+};
```

Comment:
In a lot of the code you are mixing concepts: `FLAVOR === 'enterprise'` is being submitted into `isFesUsed`. While the result works, it's really confusing. These are two different things.

`isFesUsed` is the type of property that depends on user setup, and therefore it makes sense to pass it into methods like `contactIfNeedAssistance`.

`isEnterpriseBuildUsed` is however completely constant for the whole build, and there is not much benefit in passing it down. It will always be the same for the build, and therefore can be imported directly in `Lang`. You can just do `import { FLAVOR } from './core/const.js';` and `const isEnterpriseBuild = FLAVOR === 'enterprise';`. There is no need for it to be a method, either.

Then these methods like `contactIfNeedAssistance` will have `isFesUsed` which is only be set **if fes is actually used**. Inside of the method, there will be `isFesUsed || isEnterpriseBuild` and then everything is clear.

For lang sentences where we don't have the context to know if `isFesUsed`, only `isEnterpriseBuild` will be used, and then these sentences don't need any input arguments, it will decided directly in `lang` file.

Reply:
Thank you for these - it really helps.  I will make sure to pass an argument that is really for a specific parameter and tried not to mix the two concepts.